### ISSUE TITLE
Rest parameters in arrow functions are labelled as not defined

### ISSUE THREAD

**Issue by quarterto** [jshint/jshint#1657]
Works fine:

```javascript
f = (a) => a
```
```javascript
f = function(...a) {return a}
```

Doesn't work:
```javascript
f = (...a) => a
        ^ 'a' is not defined.

f = (...a) => a
              ^ 'a' is not defined. 
```

**Comment by joesepi** [comment]
+1

**Comment by jugglinmike** [comment]
Good news! I implemented a fix for this in gh-1989. That the fix landed at dc48781adfd9626be9453a88f209ee1fad23acf4 and was included in [the `2.6.0` release of JSHint](https://github.com/jshint/jshint/releases/tag/2.6.0).

    $ git checkout 2.6.0
    $ echo "/* jshint esnext: true, unused: false */'use strict';var f = (...a) => a;" > tmp.js
    $ ./bin/jshint tmp.js
    $ echo $?
    0

@joesepi thanks for your interest in the project! In the future, please try to include additional detail in your comment--that will help us fix the bug and re-prioritize if necessary. For now, if you update to the latest version of JSHint, you should be good to go!

**Comment by joesepi** [comment]
Great. Thank you @jugglinmike. I didn't comment because I am having the same exact issue as others, but next time, I will at least say that. I will update to get the fix!
